clientupdate/clientupdate.go

```diff
@@ -300,6 +304,14 @@ func parseSynoinfo(path string) (string, error) {
 }
 
 func (up *updater) updateDebLike() error {
+	if err := requireRoot(); err != nil {
+		return err
+	}
+	if err := exec.Command("dpkg", "--status", "tailscale").Run(); err != nil && isExitError(err) {
+		// Tailscale was not installed via apt, update via tarball download
```

Comment:
These comments don't match what `updateLinuxBinary` does. Are you going to change `updateLinuxBinary` in a later PR?

Reply:
Yep, the next PR will do the tarball download from pkgs.tailcale.com and binary replacement.
I like small incremental PRs :)

Reply:
Follow-up PR for `updateLinuxBinary`: https://github.com/tailscale/tailscale/pull/9144

Reply:
Agreed re: small, incremental PRs!